pyabc/distance/scales.py

```diff
@@ -19,113 +19,134 @@
 
 Here, "only distance to observation" means that the in-sample variation
 is not taken into account.
+
+For AdaptiveAggregatedDistance, for which the data are not summary statistic
+values, but distance values, instead use either of
+* span
+* mean
+* median
 """
 
 
 import numpy as np
 
 
-def median_absolute_deviation(**kwargs):
+def median_absolute_deviation(data, **kwargs):
     """
     Calculate the sample `median absolute deviation (MAD)
     <https://en.wikipedia.org/wiki/Median_absolute_deviation/>`_
     from the median, defined as
     median(abs(data - median(data)).
     """
-    data = np.asarray(kwargs['data'])
+    data = np.array(data)
     mad = np.median(np.abs(data - np.median(data)))
     return mad
 
 
-def mean_absolute_deviation(**kwargs):
+def mean_absolute_deviation(data, **kwargs):
     """
     Calculate the mean absolute deviation from the mean.
     """
-    data = np.asarray(kwargs['data'])
+    data = np.array(data)
     mad = np.mean(np.abs(data - np.mean(data)))
     return mad
 
 
-def standard_deviation(**kwargs):
+def standard_deviation(data, **kwargs):
     """
     Calculate the sample `standard deviation (SD)
     <https://en.wikipedia.org/wiki/Standard_deviation/>`_.
     """
-    data = np.asarray(kwargs['data'])
     std = np.std(data)
```

Comment:
Regarding std calculation: by default in `np.std` the argument `ddof` is set to zero. Don't we prefer to have unbiased estimation of the std?
Details in the notes section https://docs.scipy.org/doc/numpy/reference/generated/numpy.std.html